src/python/deepgnn/pytorch/modeling/base_model.py

```diff
@@ -20,25 +20,25 @@ class BaseModel(nn.Module):
 
     def __init__(
         self,
-        feature_type: FeatureType,
+        dtype: np.dtype,
```

Comment:
Please, keep original argument names, they are more descriptive. It is weird to see  label_id and label_type, but feature_id and dtype combinations for supervised models.